packages/vite/src/node/server/middlewares/proxy.ts

```diff
@@ -37,8 +37,8 @@ export function proxyMiddleware(
   // lazy require only when proxy is used
   const proxies: Record<string, [HttpProxy.Server, ProxyOptions]> = {}
 
-  for(let context in options) {
-    if(!options[context]) continue
+  for (const context in options) {
+    if (!options[context]) continue
```

Comment:
We don't need to change the for loop, you can return as suggested by @sapphi-red. Better to keep the PR focused on the needed guard
```
  Object.keys(options).forEach((context) => {
    if (!context) {
      return
    }
    ...
```

Reply:
OK, I'll try it